ansible/roles/hetzner/create_server/tasks/main.yml

```diff
@@ -64,18 +64,12 @@
     state: absent
   when: "hcloud_server_create_res.rc == 0"
 
-#- name: "Add the server to the known_hosts file"
-#  known_hosts:
-#    path: "{{ user_home + '/.ssh/known_hosts' }}"
-#    name: "{{ ip_address }}"
-#    key: "{{ ip_address }} {{ ssh_public_key }}"
-#    state: present
-#  when: "hcloud_server_create_res.rc == 0"
-
-- name: "Add server key to local authorized keys"
```

Comment:
Why is it now commented ? Was the goal to add to the laptop/machine of the user working locally the SSH key of the VM created ?

Reply:
It doesn't work and I don't know how to make it work. 

That task is adding an `ssh-rsa`, which is the one I'm creating locally, key but the server is sending a `ecdsa-sha2-nistp256` key and since they're different the connection is blocked. Without adding the server automatically to the known_hosts file that is requestes upon the first login.

```
TASK [ping hosts] ***********************************************************************************************************************************************************************
Thursday 25 February 2021  11:55:06 +0100 (0:00:00.038)       0:00:19.093 ***** 
The authenticity of host 'xxx.xxx.xxx.xxx (xxx.xxx.xxx.xxx)' can't be established.
ECDSA key fingerprint is SHA256:xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx.
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
```

Reply:
> It doesn't work and I don't know how to make it work.

If it do not work, why did you included this task in a previous commit ?

Reply:
because I must keep investigating how to obtain that key.

Reply:
Dont forget please to create a ticket then ? @jacobdotcosta